src/components/CohortBuilder/Summary/Cards/StudiesChart.js

```diff
@@ -82,36 +96,16 @@ export const studiesQuery = (sqon) => ({
       };
     });
 
-    return Object.entries(studyLabelToCounts).reduce((accumulator, [label, counts]) => {
-      const { familyMembers, probands } = counts;
-      return [...accumulator, { label, familyMembers, probands, id: toKebabCase(label) }];
-    }, []);
+    return {
+      data: Object.entries(studyLabelToCounts).reduce((accumulator, [label, counts]) => {
+        const { familyMembers, probands } = counts;
+        return [...accumulator, { label, familyMembers, probands, id: toKebabCase(label) }];
+      }, []),
+      dictionary: [...studyDictionaryProbands, ...studyDictionaryFamily],
```

Comment:
It can be viewed as a dictionary indeed. However, I don't think the name is meaningful. I would name it differently studyToCounts or something like that.